internal/build.go

```diff
@@ -75,3 +82,31 @@ func buildCommandLoaderFilter(bundleMode bool, ignore []string) func(string, os.
 		return util.LoaderFilter{Ignore: ignore}.Apply(abspath, info, depth)
 	}
 }
+
+func assertUniqueRuleIds(paths []string) error {
+	rules, err := util.RetrieveRules(paths)
+	if err != nil {
+		return err
+	}
+
+	pathsByVisitedRuleIds := make(map[string]string)
+
+	for _, rule := range rules {
+		rule.PublicId = strings.ToUpper(rule.PublicId)
+		if _, ok := pathsByVisitedRuleIds[rule.PublicId]; ok {
+			return fmt.Errorf(
+				"We cannot create a bundle for your custom rules."+
```

Comment:
Reminded to check with Esra if we have agreed on this being the final output.
This is what we get:
![Screenshot 2021-12-31 at 17 38 10](https://user-images.githubusercontent.com/81559517/147834243-0f78be0b-825d-4322-a288-955a38bdaf5b.png)

Reply:
I've contacted her about this and will verify that this aligns with what was agreed upon.
I will also post an update on the matter here, once I have an answer.

Reply:
According to [Esra's comment on CFG-1233](https://snyksec.atlassian.net/browse/CFG-1233?focusedCommentId=256581), the agreed-upon error message is:
```
We cannot create a bundle for your custom rules.
The bundle contains duplicate rules.
Please ensure all rules have unique public IDs.

Duplicate rules are:
- ./rules/CUSTOM-RULE-10/main.rego
- ./rules/CUSTOM-RULE-11/main.rego
```